Management/src/main/webapp/WEB-INF/jsp/product.jsp

```diff
@@ -0,0 +1,21 @@
+<%@include file="commons/header.jspf"%>
+<%@include file="commons/navigation.jspf"%>
+<div class="container">
+	<form:form method="post" commandName="product">
+		<fieldset class="form-group">
+			<form:label path="name">Name:</form:label>
+			<form:input path="name" type="text" class="form-control"
+				required="required" />
+				<form:label path="name">Model:</form:label>
+			<form:input path="model" type="text" class="form-control"
+				required="required" />
+				<form:label path="name">Price:</form:label>
+			<form:input path="price" type="text" class="form-control"
+				required="required" />
+			<form:errors path="name" cssClass="text-warning"></form:errors>
```

Comment:
Is not better to add the error under the `name` field?

Reply:
done